server/src/services/organizations.service.ts

```diff
@@ -112,10 +112,43 @@ export class OrganizationsService {
     }, manager);
   }
 
-  async fetchUsers(user: any): Promise<FetchUserResponse[]> {
+  async fetchUsersByValue(user: any, searchInput: string): Promise<any> {
+    const organizationUsers = await createQueryBuilder(OrganizationUser, 'organization_user')
+      .innerJoinAndSelect('organization_user.user', 'user')
+      .where('organization_user.organization_id = :organizationId', {
+        organizationId: user.organizationId,
+      })
+      .andWhere(
+        new Brackets((qb) => {
+          qb.where('user.email like :email', {
+            email: `%${searchInput}%`,
+          })
+            .orWhere('user.firstName like :firstName', {
+              firstName: `%${searchInput}%`,
+            })
+            .orWhere('user.lastName like :lastName', { lastName: `%${searchInput}%` });
+        })
+      )
+      .getMany();
+
+    return organizationUsers?.map((orgUser) => {
+      return {
+        email: orgUser.user.email,
+        firstName: orgUser.user.firstName,
+        lastName: orgUser.user.lastName,
+        name: `${orgUser.user.firstName} ${orgUser.user.lastName}`,
+        id: orgUser.id,
+        userId: orgUser.user.id,
+      };
+    });
+  }
+
+  async fetchUsers(user: any, page: number, options: any): Promise<FetchUserResponse[]> {
     const organizationUsers = await this.organizationUsersRepository.find({
-      where: { organizationId: user.organizationId },
+      where: { organizationId: user.organizationId, user: options },
       relations: ['user'],
+      take: page ? 10 : undefined,
+      skip: page ? 10 * (page - 1) : undefined,
     });
 
     const isAdmin = await this.usersService.hasGroup(user, 'admin');
```

Comment:
This function is used by only admin now, do we need this check?

Reply:
if then, we can remove it